core/modules/util/Error.cc

```diff
@@ -0,0 +1,56 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2014 LSST Corporation.
```

Comment:
Ok thanks. Does the Copyright needs to be updated each year on all source files? If yes we should template this, isn't it?